chipsec/hal/common/vmm.py

```diff
@@ -151,6 +152,9 @@ def dump_device(self) -> None:
         bars = self.cs.hals.Pci.get_device_bars(self.bus, self.dev, self.fun)
         for (bar, isMMIO, _, _, _, size) in bars:
             if isMMIO:
-                self.cs.hals.MMIO.dump_MMIO(bar, size)
+                self.cs.hals.Mmio.dump_MMIO(bar, size)
             else:
-                self.cs.hals.PortIO.dump_IO(bar, size, 4)
+                self.cs.hals.Io.dump_IO(bar, size, 4)
```

Comment:
I changed this to `dump_range`
```suggestion
                self.cs.hals.Io.dump_range(bar, size, 4)
```